pkg/dev/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:edge
```

Comment:
I think it would be better to inherit from the exiting ssh example instead of replicating the setup